input/postgres/relation_stats.go

```diff
@@ -65,6 +66,44 @@ SELECT c.oid,
 			 AND c.oid NOT IN (SELECT pd.objid FROM pg_catalog.pg_depend pd WHERE pd.deptype = 'e' AND pd.classid = 'pg_catalog.pg_class'::regclass)
 			 AND n.nspname NOT IN ('pg_catalog','pg_toast','information_schema')
 			 AND ($1 = '' OR (n.nspname || '.' || c.relname) !~* $1)
+ UNION
```

Comment:
Probably not a huge deal, but I think these could all be `UNION ALL`s, no?

Reply:
Good point - I'll adjust to UNION ALL.